vimrc

```diff
@@ -221,3 +221,4 @@ else
   let &t_EI = "\<Esc>]50;CursorShape=0\x7"
 end
 
+set shell=/bin/sh
```

Comment:
this is resolving the error that running specs from vim while in tmate attempts to use an incorrect ruby version

this is the suggested rvm solution for vim (https://rvm.io/integration/vim)

there's no problem using /bin/sh as our vim shell. we don't use anything zsh-y in our vim config